openmetadata-docs/content/deployment/upgrade/docker.md

```diff
@@ -22,39 +22,104 @@ It is adviced to go through [openmetadata release notes](/deployment/upgrade#bre
+3. Start OpenMetadata with the 0.13.1 release.
+
+These steps are based on MySQL, but the steps are analogous for Postgres.
+
+### Backup 0.12.3 data
+
+1. Get inside the `openmetadata_ingestion` container with `docker exec -it openmetadata_ingestion bash`.
+2. Create a virtual environment to install an upgraded `metadata` version to run the backup command:
+   1. `python -m venv venv`
+   2. `source venv/bin/activate`
+   3. `PIP_USER=false pip install openmetadata-ingestion~=0.13.1`
+3. Validate the installed `metadata` version with `python -m metadata --version`, which should tell us that we are
+    indeed at 0.13.1. Notice the `python -m metadata` vs. `metadata`. As we're inside the container, the raw metadata 
+    command will use the system-wide one, which is at 0.12.3.
+4. Run the backup using the updated `metadata` CLI:
+    ```
+    python -m metadata backup -u openmetadata_user -p openmetadata_password -H mysql -d openmetadata_db --port 3306
+    ```
+   if using Postgres:
+    ```
+    python -m metadata backup -u openmetadata_user -p openmetadata_password -H postgresql -d openmetadata_db --port 5432 -s public
+    ```
+5. From outside the container, copy the backup file to safety. For example `docker cp openmetadata_ingestion:/opt/airflow/openmetadata_202212201528_backup.sql .`
+    In our case, the backup file was named `openmetadata_202212201528_backup.sql`. You can copy the name from the backup
+    command output.
+
+### Add volumes and port mapping
+
+1. Stop docker compose
+2. Update the `mysql` service in the compose file so that it looks like:
+    ```
+    mysql:
+      container_name: openmetadata_mysql
+      image: openmetadata/db:0.12.3
+      restart: always
+      environment:
+        MYSQL_ROOT_PASSWORD: password
+      expose:
+        - 3306
+      ports:
+        - "3306:3306"
+      volumes:
+       - ./docker-volume/db-data:/var/lib/mysql
+      networks:
+        - app_net
+      healthcheck:
+        test: mysql --user=root --password=$$MYSQL_ROOT_PASSWORD --silent --execute "use openmetadata_db"
+        interval: 15s
+        timeout: 10s
+        retries: 10 
+    ```
+   Note how we added the `ports` and `volumes` sections. If you're using postgres, these would look like:
+    ```
     ports:
-      - "3306:3306"
-    ...
-```
-
-### 5. Stop, Remove and Start your Containers
-Stop and remove (without deleting volumes) the already running containers. Then run the `docker compose up -d` command on the new compose file.
+      - "5432:5432"
+    volumes:
+     - ./docker-volume/db-data-postgres:/var/lib/postgresql/data
+    ```
+3. Start docker compose with `docker compose up` on the updated file.
+   1. This will run a clean database instance with ports and volumes.
+   2. The OpenMetadata server will run the 0.12.3 migrations to populate the starting tables.
+
+### Restore the backup
+
+1. On your laptop/VM, prepare a virtual environment and install `openmetadata-ingestion==0.13.1`
```

Comment:
I think we should be more specific that we want to run this command for a place where the mysql database running in the container is accessible and maybe add that we will be using our backup to repopulate the database.